### PR TITLE
Bookstore: connect to api

### PR DESCRIPTION
**Pull request by adarachel**
Make use of the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as your backend.
Use [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make your HTTP requests to the backend
Fetch your books from the Bookstore API after the component has rendered
Refactor your add book and remove book features to persist your changes in the server
Your application should behave the same way after the refactoring

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,40 +1,60 @@
...
-    },
+    addBook: (state, action) => ({
+      ...state,
+      books: [...state.books, action.payload],
+    }),
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter((book) => book.item_id !== action.payload),
+    }),
+  },
```
Comment (iambenkis) [inline src/redux/books/booksSlice.js:36]: - [ ] Good job connecting your bookstore to an API, 👏 , I noticed that the remove button is not working as it should , Kindly fix that to go to the next step 👍 

Capture 1

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @adarachel 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi @adarachel,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Add and remove functionality works as expected 👍 
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adarachel** [comment]
Hello @huseyinbiyik !! Please i am in need of a code review for my project: [bookstore: add styling](github.com/adarachel/project-bookstore/pull/5)

Please I am desperate. The deadline is in one hour. I have requested for a review over 50 minutes ago, but no code reviewer has reviewed my pull request. I am worried that it will not be approved on time. Please help me review the pull request, thank you.

**Comment by adarachel** [comment]
Hello @iambenkis !! Please i am in need of a code review for my project: [bookstore: add styling](github.com/adarachel/project-bookstore/pull/5)

Please I am desperate. The deadline is in one hour. I have requested for a review over 50 minutes ago, but no code reviewer has reviewed my pull request. I am worried that it will not be approved on time. Please help me review the pull request, thank you.

**Pull request merged**